### PR TITLE
cognitive maps blogpost

### PR DESCRIPTION
**Pull request by gregbunce**
This is WIP pull request to see how things look.  I'll put more time into it tomorrow to fix the needed areas.

[article preview](deploy-preview-1324--gis-utah-gov.netlify.com/cognitive-maps/)

### CODE REVIEW

**Comment by steveoh** [comment]
Let me know when you would like me to review this by removing the draft status!

File: _posts/2020-01-29-cognitive-maps.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+### Organizing Life's Experiences
+{: .text-left}
+
+We now know that our brain is a spatial organ. The hippocampus, in particular, is responsible for mapping, navigation, and memory storage. These functions are linked to additional mechanisms such as episodic and long term memories. Research has shown that when we exercise these functions we see an increase in brain activity. These active neurons not only create new connections but preserve and organize existing ones (Javadi et al 2017).
+
+At the core of these functions is a system that stores and retrieves spatial information. In this context, it's easy to make a comparison between the brain a GIS. The hippocampus acts much like a file indexing system working with other parts of the brain, that function as a database, making transactions. When we add the episodic memory aspect, it's similar to enabling the spatial component on the database: memories now contain a geographic location. This explains why places, --such as a bend in the road-- or an event --such as September 11th-- can trigger the retrieval of certain memories. When we think of September 11th, 2001, we not only remember the event but also the where-we-were.
```
Comment (steveoh) [inline _posts/2020-01-29-cognitive-maps.md]: should a be and in the sentence `In this context, it's easy to make a comparison between the brain a GIS.`?

```suggestion
At the core of these functions is a system that stores and retrieves spatial information. In this context, it's easy to make a comparison between the brain and GIS. The hippocampus acts much like a file indexing system working with other parts of the brain, that function as a database, making transactions. When we add the episodic memory aspect, it's similar to enabling the spatial component on the database: memories now contain a geographic location. This explains why places, --such as a bend in the road-- or an event --such as September 11th-- can trigger the retrieval of certain memories. When we think of September 11th, 2001, we not only remember the event but also the where-we-were.
```

File: _posts/2020-01-29-cognitive-maps.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  - Featured
+tags:
+  - education
+  - maps
+---
+
+![Network Head]({% link images/network_head.png %}){: .flex .flex--center}
+
+I bet you remember where you were on the morning of September 11th, 2001 when you heard the news. I’d venture to say that you also remember the city, state, and room that you were standing in. But why is this? Why is it that certain memories are permanently tied to a geographic location?
+
```
Comment (steveoh) [inline _posts/2020-01-29-cognitive-maps.md]: I also might suggest that we move the image down a paragraph so the article blurb isn't empty...

image

**Comment by steveoh** [comment]
I think we are missing the Philos 2009 attribution...

File: _posts/2020-01-29-cognitive-maps.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+<div class="flex flex--around">
+  <figure class="caption">
+    <img class="caption__image" src="{% link images/london_iconic_cabs.png %}" alt="London’s Black Cabs" />
+    <figcaption class="caption__text">London’s Iconic Black Cabs (<a href="https://www.ft.com/content/ccdde2f0-c0d0-11e4-876d-00144feab7de">Financial Times</a>)</figcaption>
+  </figure>
+</div>
+
+Additional brain navigation studies were conducted using a highly accurate and interactive virtual reality (VR) simulation of London’s central city as both cab drivers and non-cab drivers navigated the landscape using the Sony Playstation video game, The Gateway (Philos 2009). The game’s streets network was created using map data from the British Ordnance Survey and also simulated entities such as traffic signals, traffic, and pedestrians. Functional MRI (fMRI) scans taken while they navigated around VR London showed that brain activity in the hippocampus was active in both the licensed cab drivers and the non-cab drivers.
```
Comment (steveoh) [inline _posts/2020-01-29-cognitive-maps.md]: I dont' see a philos 2009 source link at the bottom...

File: _posts/2020-01-29-cognitive-maps.md (hunk 2 above)
Comment (gregbunce) [reply _posts/2020-01-29-cognitive-maps.md]: good point.  i'll fix it.

File: _posts/2020-01-29-cognitive-maps.md (hunk 3 above)
Comment (gregbunce) [reply _posts/2020-01-29-cognitive-maps.md]: good catch.  i'll add it.

**Comment by gregbunce** [comment]
thanks for reviewing this, @steveoh !  I made the chnages.  Also, I tired to merge the footnotes pr, but I don't think i did it right.  let me know the correct way.  thanks again!

**Pull request merged**